docker-compose.override.yml

```diff
@@ -18,7 +18,7 @@ services:
       API_DEBUG: 1
       ENVIRONMENT: local
       LOGGING_LEVEL: INFO
-      JOB_SERVICE_ENABLED: 1
+      JOB_SERVICE_ENABLED: 0
```

Comment:
should this be turned off as default? We might want to have the job api functionality on by default?